### PR TITLE
docs: update state sync

### PR DESCRIPTION
**Pull request by yaruwangway**

### CODE REVIEW

**Comment by yaruwangway** [comment]
hi , can we use rpc_server not maintained by us in our official state-sync tutorial ? @okwme 
`https://cosmos-rpc.polkachu.com:443,https://rpc-cosmoshub-ia.cosmosia.notional.ventures:443,https://rpc.cosmos.network:443`

**Comment by okwme** [comment]
i think that's no problem @yaruwangway !

File: docs/getting-started/quickstart.md [hunk 1]
```diff
@@ -32,13 +32,13 @@ gaiad init CUSTOM_MONIKER --chain-id cosmoshub-4
 
 #Set minimum gas price & peers
 sed -i'' 's/minimum-gas-prices = ""/minimum-gas-prices = "0.0025uatom"/' $HOME/.gaia/config/app.toml
-sed -i'' 's/persistent_peers = ""/persistent_peers = "6e08b23315a9f0e1b23c7ed847934f7d6f848c8b@165.232.156.86:26656,ee27245d88c632a556cf72cc7f3587380c09b469@45.79.249.253:26656,538ebe0086f0f5e9ca922dae0462cc87e22f0a50@34.122.34.67:26656,d3209b9f88eec64f10555a11ecbf797bb0fa29f4@34.125.169.233:26656,bdc2c3d410ca7731411b7e46a252012323fbbf37@34.83.209.166:26656,585794737e6b318957088e645e17c0669f3b11fc@54.160.123.34:26656,5b4ed476e01c49b23851258d867cc0cfc0c10e58@206.189.4.227:26656"/' $HOME/.gaia/config/config.toml
+sed -i'' 's/persistent_peers = ""/persistent_peers = '"\"$(curl -s https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/chain.json | jq -r '[foreach .peers.seeds[] as $item (""; "\($item.id)@\($item.address)")] | join(",")')\""'/' $HOME/.gaia/config/config.toml
```
Comment (okwme) [inline docs/getting-started/quickstart.md:35]: wow `jq` super powers 💪

**Review by okwme — APPROVED** [review]
uACK

**Pull request merged**